web/packages/design/src/theme/lightTheme.ts

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
(commenting on the whole file seems to be broken so I'm leaving a comment here)

Is it fine if I keep reporting some minor issues with the new theme here?

The focus state on the passwordless button in the login modal is barely noticeable in the light theme and pretty much nonexistent in Connect. You can quickly compare current version with v12 by attempting to log in to teleport-12-ent, either through the Web UI or Connect.

![passwordless button](https://user-images.githubusercontent.com/27113/233112593-948d5e4e-3742-475a-b9cd-4c047f88d103.png)

Reply:
Fixed in https://github.com/gravitational/teleport/pull/25119